src/cls/cmpxattr/client.cc

```diff
@@ -0,0 +1,43 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab ft=cpp
+
+/*
+ * Ceph - scalable distributed file system
+ *
+ * Copyright (C) 2024 IBM Corp.
+ *
+ * This is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License version 2.1, as published by the Free Software
+ * Foundation.  See file COPYING.
+ */
+
+#include "include/rados/librados.hpp"
+#include "client.h"
+#include "ops.h"
+
+namespace cls::cmpxattr {
+  int cmp_vals_set_vals(librados::ObjectWriteOperation& op,
+			Mode mode, Op comparison,
+			const ComparisonMap& cmp_pairs,
```

Comment:
if you move the by-ref parameters, it would be safer to pass as `&&` instead.
so, the caller know to avoid use-after-move.